main.go

```diff
@@ -1,12 +1,29 @@
 package main
 
 import (
+	"context"
+	"flag"
+	"log"
+
 	"github.com/hashicorp/terraform-plugin-sdk/v2/plugin"
 	"github.com/sullivtr/terraform-provider-graphql/graphql"
 )
 
 func main() {
-	plugin.Serve(&plugin.ServeOpts{
-		ProviderFunc: graphql.Provider,
-	})
+	var debugMode bool
```

Comment:
These changes are also in https://github.com/sullivtr/terraform-provider-graphql/pull/58, I'll rebase this branch once the other PR is merged.

Reply:
Go ahead and rebase. I merged #58. I haven’t cut it into a release yet. I have some other small ancillary changes in mind I need to make so I’ll try to fit those into one release.

Reply:
Rebased on master.